src/mappings/index.ts

```diff
@@ -156,6 +158,10 @@ export async function handleTokenCreate(context: Context): Promise<void> {
 }
 
 export async function handleTokenTransfer(context: Context): Promise<void> {
+  if (context.extrinsic && [Extrinsic.acceptOffer, Extrinsic.buy].includes(context.extrinsic?.name as Extrinsic)) {
+    return;
+  }
+
```

Comment:
```suggestion
  if (context.extrinsic && [Extrinsic.acceptOffer, Extrinsic.buy].includes(context.extrinsic?.name as Extrinsic)) {
    logger.info(`[SEND] SKIP: ${context.event.blockNumber}, because of ${context.extrinsic?.name}`);
    return;
  }

```